README.md

```diff
@@ -52,14 +52,14 @@ For details, see the [Developers site](https://developer.trustwallet.com/add_new
 
 There are several scripts available for maintainers:
 
-- `npm run check` -- Execute validation checks; also used in continuous integration.
-- `npm run check-sanity` -- Strict subset of checks
-- `npm run fix` -- Perform automatic fixes where possible
-- `npm run fix-sanity` -- Stricter subset
-- `npm run updateAuto` -- Run automatic updates from external sources, executed regularly (GitHub action)
-- `npm run update` -- Run manual updates from external sources, for manual use.
-- `npm test` -- Run script unit tests
-- `npm lint` -- Run Lint static code check
+- ✅`npm run check` -- Execute validation checks; also used in continuous integration.
```

Comment:
- ✅`npm run check-sanity` -- Strict subset of checks
 - ✅`npm run fix` -- Perform automatic fixes where possible
 - ✅`npm run fix-sanity` -- Stricter subset
 - ✅`npm run updateAuto` -- Run automatic updates from external sources, executed regularly (GitHub action)
 - ✅ `npm run update` -- Run manual updates from external sources, for manual use.
 - ✅`npm test` -- Run script unit tests
 - ✅`npm lint` -- Run Lint static code check